api/test/search.test.js

```diff
@@ -43,4 +43,23 @@ describe('search', () => {
     expect(firstGene.name).toBe('POLR3F');
     expect(firstMetabolite.name).toMatch(/Asparaginyl-Cysteinyl/);
   });
-});
+  
+  test('model search with special characters should not fail', async () => {
+    const [data1, data2] = await Promise.all([
+      search({
+        searchTerm: 'rna/dna: (met)',
+        model: 'HumanGem',
+        version: HUMAN_GEM_VERSION,
+      }),
+      search({
+        searchTerm: 'cy\\|+ {zinc}! glyco/mg',
+        model: 'HumanGem',
+        version: HUMAN_GEM_VERSION,
+      }),
+    ]);
+    for (const data of [data1, data2]) {
+      const { name } = data['Human-GEM'];
+      expect(name).toEqual('Human-GEM');
+    }
+  });
+});
```

Comment:
Add newline?

Reply:
Not quite sure. It's not added in the rest of testing js files.

Reply:
`ma-exec api yarn format` does react to this as well so I think it makes sense to add it

Reply:
It seems there is a prettier issue as @MalinAhlberg also pointed out. I guess this is caused when I solving the merging conflict in the browser, which ignored the prettier.